images/singleuser/Dockerfile

```diff
@@ -60,6 +60,7 @@ RUN apt-get update --yes && \
         r-base-dev \
         r-cran-littler \
         openjdk-11-jdk \
+        julia \
```

Comment:
I would also strongly recommend installing Julia by just downloading their binaries (https://github.com/berkeley-dsep-infra/datahub/blob/staging/deployments/julia/image/install-julia.bash) rather than via apt. apt's julia is outdated and not well maintained, so the traditional advantage of security patches doesn't even apply I think.